apps/dotcom/client/src/tla/app/zero-polyfill.ts

```diff
@@ -153,7 +153,7 @@ export class Zero {
 					{
 						table: 'file_state',
 						event: 'insert',
-						row: { fileId: data.id, userId: store.user.id } as any,
+						row: { fileId: data.id, userId: store.user.id, firstVisitAt: Date.now() } as any,
```

Comment:
We didn't set the `firstVisitAt` when we created the file, so it never got populated (`firstVisitAt` is marked as a immutable column).